src/style.css

```diff
@@ -1 +1,37 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-of-type(-n + 3) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-of-type(-n + 4) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-of-type(-n + 6) {
+  background-image: url(/src/images/star-active.svg);
+}
```

Comment:
There seems to be a typo in the selector `.stars--5 .stars__star:nth-of-type(-n + 6)`. It should be `-n + 5` to match exactly five stars. Please correct it to `.stars--5 .stars__star:nth-of-type(-n + 5)`.